lib/mpl_toolkits/mplot3d/proj3d.py

```diff
@@ -124,6 +124,15 @@ def persp_transformation(zfront, zback):
                      [0,0,-1,0]
                      ])
 
+def ortho_transformation(zfront, zback):
+    a = -(zfront + zback)
+    b = -(zfront - zback)
+    return np.array([[2,0,0,0],
+                     [0,2,0,0],
+                     [0,0,-2,0],
+                     [0,0,a,b]
+                     ])
```

Comment:
Can you provide a reference for this formulation? I am not an expert in 3d transforms, but this doesn't match anything I am seeing on the wikipedia page for orthographic projection.